CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKFPSRenderable.java

```diff
@@ -790,28 +777,18 @@ private int UpdateUniformBuffers(MemoryStack stack) {
         vertexUBO.visibilityLow = parent.getDisplayCamera().getVisibilityLow();
         vertexUBO.visibilityHigh = parent.getDisplayCamera().getVisibilityHigh();
         
+        // Update the push constants data
+        vertexPushConstants.clear();
```

Comment:
Not sure that's needed.

Reply:
True not needed as we do a flip() after, removed